crates/forge/tests/e2e/common/runner.rs

```diff
@@ -48,3 +49,18 @@ pub(crate) fn setup_package(package_name: &str) -> TempDir {
 
     temp
 }
+
+pub(crate) fn gen_current_branch() -> String {
+    let name: &str = "BRANCH_NAME";
+    match env::var(name) {
+        Ok(v) => v,
+        Err(_e) => {
+            let output = SnapboxCommand::new("git")
```

Comment:
`SnapboxCommand` is mostly designed for testing. Just `Command` would be fine here.

Reply:
SnapboxCommand is [alias](https://github.com/foundry-rs/starknet-foundry/pull/477/files/0e9c62e8607318f8dbd088f50a10633e875e719b#diff-56dbdcd8d13ff4a2f6a482083b4e3fdadf7d8f0d6246aa8998841eb33cb826e7R5) of Command. It was import before. Should I remove this alias?

Reply:
I mean using Command from std directly. https://doc.rust-lang.org/std/process/struct.Command.html

It's a bit different structure